tests/screenshot/apps/components/VirtualList.js

```diff
@@ -0,0 +1,25 @@
+import Item from '../../../../Item';
+import VirtualList from '../../../../VirtualList';
+import ri from '@enact/ui/resolution';
+import React from 'react';
+
+const renderItem = () => ({index, ...rest}) => { // eslint-disable-line enact/display-name, enact/prop-types
+	const items = ['Item 000', 'Item 001', 'Item 002', 'Item 003', 'Item 004', 'Item 005', 'Item 006', 'Item 007', 'Item 008', 'Item 009'];
+	const itemStyle = {
+		borderBottom: ri.unit(6, 'rem') + ' solid #202328',
+		boxSizing: 'border-box',
+		height: ri.unit(144, 'rem')
+	};
+
+	return (
+		<Item {...rest} style={itemStyle}>
+			{items[index]}
+		</Item>
+	);
+};
+
+const VirtualListTests = [
+	<VirtualList dataSize={10} itemSize={ri.scale( 144)} itemRenderer={renderItem(ri.scale(144))} />
```

Comment:
The scrollbar is not displayed and the right side is cut off.
Either it's a VirtualList issue, or you might need a parent div.

After build, you can display the screen by ` serve tests/screenshot/dist`
If serve is not installed, you need to install it.
CC @stanca-pop-lgp

Reply:
I added an extra VirtualList with verticalScrollbar and horizontalScrollbar to be visible